Classes/Repository/RepositoryViewController.swift

```diff
@@ -159,6 +159,20 @@ ContextMenuDelegate {
             .newIssue(issueController: newIssueViewController)
     }
 
+    func workingCopyAction() -> UIAlertAction? {
+        guard let remote = self.repoUrl.absoluteString.addingPercentEncoding(withAllowedCharacters: CharacterSet.alphanumerics) else { return nil}
+
+        guard let url = URL(string: "working-copy://show?remote=\(remote)") else { return nil }
+        guard UIApplication.shared.canOpenURL(url) else { return nil }
```

Comment:
@palmin Would this work without adding `working-copy` to `LSApplicationQueriesSchemes` ? 

https://developer.apple.com/documentation/uikit/uiapplication/1622952-canopenurl

> If your app is linked on or after iOS 9.0, you must declare the URL schemes you pass to this method by adding the LSApplicationQueriesSchemes key to your app's Info.plist file. This method always returns false for undeclared schemes, whether or not an appropriate app is installed. To learn more about the key, see LSApplicationQueriesSchemes.